src/viam/examples/modules/complex/proto/buf.gen.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Viam Inc.
```

Comment:
I don't think we'll need any `buf.*` files once we figure out how to put proto generation into cmake.